website/docs/docs/deploy/webhooks.md

```diff
@@ -132,5 +132,379 @@ An example of a webhook payload for an errored run:
 }
 ```
 
+## API for webhooks {#api-for-webhooks}
+You can use the dbt Cloud API to create new webhooks that you want to subscribe to, get detailed information about your webhooks, and to manage the webhooks that are associated with your account. The following sections describe the API endpoints you can use for this. 
+
+### List all webhook subscriptions
+<details>
```

Comment:
i'm experimenting with the collapsible section component for the API docs. the table and the color doesn't go well together imho. leaning towards removing it

Reply:
Ah, that's a fair point. I'm leaning towards removing it but will let others on your team chime in here!

Reply:
:gavel: removing it!